e2e/helper.ts

```diff
@@ -23,22 +31,41 @@ export const test = base.extend<AppOptions & AppFixtures>({
   emberOptions: [{ setTesting: true, mockSentry: true }, { option: true }],
   clock: [
     async ({ page, clockOptions }, use) => {
+      let now = clockOptions.now;
+      if (typeof now === 'string') {
+        now = Date.parse(now);
+      }
+
+      let pwClock = pwFakeTimers.install({
+        ...clockOptions,
+        now,
+        toFake: ['Date'],
+      });
+
```

Comment:
Does this differ from `FakeTimers` in `e2e/fixtures/fake-timers`? If it does, should we move the change into `e2e/fixtures/fake-timers`? It seems they are both trying to mock the same thing using `@sinonjs/fake-timers`.

Reply:
one is mocking on the browser side, the other on the Playwright side. previously the mock API was running on the browser side too, so we only had to mock the timers on one side.

Reply:
I guess we could also migrate these parts to use Playwright's newly added clock API.
Refs:
- https://playwright.dev/docs/clock
- https://playwright.dev/docs/api/class-clock#methods

Reply:
yeah, that sounds reasonable 👍 

okay if we move that to a follow-up PR?

Reply:
I'm fine with it being in a follow-up PR!